extensions/replication/replicationStatusProcessor/ReplicationStatusProcessor.js

```diff
@@ -204,14 +204,16 @@ class ReplicationStatusProcessor {
      *   in PEM format
      * @param {Object} mConfig - metrics config
      * @param {String} mConfig.topic - metrics config kafka topic
+     * @param {Boolean} enableGC - flag to enable garbage collection - Based on GC config
      */
     constructor(kafkaConfig, sourceConfig, repConfig,
-                internalHttpsConfig, mConfig) {
+                internalHttpsConfig, mConfig, enableGC) {
```

Comment:
Maybe we should just call it `gcConfig` ? `enableGC` is a bit missleading, it makes us think it's a boolean.